app/controllers/projects_controller.rb

```diff
@@ -36,7 +36,8 @@ def index
           html: render_to_string(
             partial: 'projects/index/team_projects.html.erb',
                      locals: { projects: @projects }
-          )
+          ),
+          count: @projects.count
```

Comment:
will perform additional query, please use 'size' instead as collection is already loaded